### PR TITLE
Broken Forms links

### PR DESCRIPTION
**Pull request by hartviglarsen**
The purchase and sign up link on https://docs.umbraco.com/umbraco-forms/umbraco-forms are broken. I am not sure if you want to continue to redirect the sign up to a documentation page or if it should go to the trial flow - I chose the latter :)

### CODE REVIEW

File: umbraco-forms/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ meta.Description: "Documentation on how to work with Umbraco Forms for both edit
 
 Umbraco Forms is tool that lets you build forms of all shapes and sizes, and put them on your Umbraco websites. Build forms using a long list of elements like multiple choice, dropdowns, text areas and checkboxes. Choose between a series of different workflow and control what happens once a form has been submitted.
 
-[Purchase Umbraco Forms](https://umbraco.com/apps/umbraco-forms/) or sign up for an [Umbraco Cloud](../umbraco-cloud/) project where Umbraco Forms is part of the package.
+[Purchase Umbraco Forms](https://umbraco.com/products/umbraco-forms/) or sign up for an [Umbraco Cloud](https://try.umbraco.com/) project where Umbraco Forms is part of the package.
```
Comment (sofietoft) [inline umbraco-forms/README.md:10]:
```suggestion
[Purchase Umbraco Forms](https://marketplace.umbraco.com/package/umbraco.forms) or sign up for an [Umbraco Cloud](https://try.umbraco.com/) project where Umbraco Forms is part of the package.
```

**Review by sofietoft — COMMENTED** [review]
Thanks for fixing the links @hartviglarsen ! 💪 

How about we link to the new marketplace for purchasing Forms? 🤔 
And I agree about linking directly to "try Umbraco" for the Cloud link!

File: umbraco-forms/README.md (hunk 1 above)
Comment (hartviglarsen) [reply umbraco-forms/README.md:10]: @sofietoft I definitely like the new Marketplace and it is a welcome change many people have longed for. However, if the purpose of the link is to allow someone to purchase Umbraco Forms, I am not sure linking to Marketplace is the best solution.

Umbraco.com has a fairly clear in-your-face call to action to purchase Forms, whereas Forms on Marketplace does not mention anything about purchasing.. unless one clicks "Package information" under Links, which is not the most intuitive way of getting to a _buy_ button :D 

Marketplace does look really nice, though.

Comment (sofietoft) [reply umbraco-forms/README.md:10]: That is a good point 😄

**Pull request merged**